examples/image-to-text/run_pipeline.py

```diff
@@ -119,9 +130,18 @@ def main():
     if args.prompt is None and model_type == "llava":
         args.prompt = "<image>\nUSER: What's the content of the image?\nASSISTANT:"
     elif args.prompt is None and model_type == "llava_next":
-        args.prompt = "[INST] <image>\nWhat is shown in this image? [/INST]"
-        if args.model_name_or_path in ["llava-hf/llava-v1.6-vicuna-13b-hf", "llava-hf/llava-v1.6-vicuna-7b-hf"]:
-            args.prompt = "A chat between a curious human and an artificial intelligence assistant. The assistant gives helpful, detailed, and polite answers to the human's questions. USER: <image>\nWhat is shown in this image? ASSISTANT:"
+        if args.model_name_or_path in ["llava-hf/llava-v1.6-34b-hf",]:
+            args.prompt = "<|im_start|>system\nAnswer the questions.<|im_end|><|im_start|>user\n<image>\nWhat is shown in this image?<|im_end|><|im_start|>assistant\n"
+        else:
+            processor = LlavaNextProcessor.from_pretrained(args.model_name_or_path)
+            conversation = [{
+                "role": "user",
+                "content": [
+                        {"type": "text", "text": "What is shown in this image?"},
+                        {"type": "image"},
+                ],
+            }]
+            args.prompt = processor.apply_chat_template(conversation, add_generation_prompt=True)
```

Comment:
```suggestion
        processor = LlavaNextProcessor.from_pretrained(args.model_name_or_path)
        conversation = [{
            "role": "user",
            "content": [
                    {"type": "text", "text": "What is shown in this image?"},
                    {"type": "image"},
            ],
        }]
        args.prompt = processor.apply_chat_template(conversation, add_generation_prompt=True)
```

@KimBioInfoStudio, shouldn't we remove the template for these models? I thought LLaVA models are designed to handle interactive and context-rich conversations

Reply:
Just my thought

Reply:
huggingface's latest feature use chat_template.json for chat, but not all models have this file yet, so just modified for available models at the point, seems `llava-hf/llava-v1.6-34b-hf` Processer have bug, seems they fix it 20 days ago
![image](https://github.com/user-attachments/assets/dc394b9e-6b08-4bef-80a5-062b0d8a9339)
need more validation on this model and `lava-1.5` which just add chat_template within 20 days